udacity/css/style.css

```diff
@@ -0,0 +1,221 @@
+body,
+div,
+ul,
+li,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  padding: 0;
+  margin: 0;
```

Comment:
Is declaring padding and margin 0 necessary? Also if you declare this on the body, the objects ( divs, ul, etc) inside the body tag should also have these parameters.